query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/QueryBuilder.java

```diff
@@ -470,4 +472,47 @@ public static UserDefinedType udt(@NonNull CqlIdentifier name) {
   public static UserDefinedType udt(@NonNull String name) {
     return udt(CqlIdentifier.fromCql(name));
   }
+
+  /**
+   * Creates a new {@code TRUNCATE} query.
+   *
+   * @param table the name of the table to truncate.
+   * @return the truncation query.
+   */
+  public static Truncate truncate(@NonNull CqlIdentifier table) {
+    return truncate(null, table);
+  }
+
+  /**
+   * Creates a new {@code TRUNCATE} query.
+   *
+   * @param table the name of the table to truncate.
+   * @return the truncation query.
+   */
+  public static Truncate truncate(@NonNull String table) {
+    return truncate(null, CqlIdentifier.fromCql(table));
+  }
+
+  /**
+   * Creates a new {@code TRUNCATE} query.
+   *
+   * @param keyspace the name of the keyspace to use.
+   * @param table the name of the table to truncate.
+   * @return the truncation query.
+   */
+  public static Truncate truncate(@Nullable CqlIdentifier keyspace, @NonNull CqlIdentifier table) {
+    return new DefaultTruncate(keyspace, table);
+  }
+
+  /**
+   * Creates a new {@code TRUNCATE} query.
+   *
+   * @param keyspace the name of the keyspace to use.
+   * @param table the name of the table to truncate.
+   * @return the truncation query.
+   */
+  public static Truncate truncate(@Nullable String keyspace, @NonNull String table) {
+    return truncate(
+        keyspace == null ? null : CqlIdentifier.fromCql(keyspace), CqlIdentifier.fromCql(table));
+  }
```

Comment:
👍 
Can you add a comment for the string variants to indicate that they use `CqlIdentifier.fromCql` for the conversion? See selectFrom at line 68 for example.